pkg/cmd/pr/list/list_test.go

```diff
@@ -176,39 +176,100 @@ func TestPRList_filteringAssignee(t *testing.T) {
 	}
 }
 
-func TestPRList_filteringAssigneeLabels(t *testing.T) {
+func TestPRList_bothNonDraftAndDraft(t *testing.T) {
 	http := initFakeHTTP()
 	defer http.Verify(t)
 
-	_, err := runCommand(http, true, `-l one,two -a hubot`)
-	if err == nil && err.Error() != "multiple labels with --assignee are not supported" {
```

Comment:
I removed this test, because it seems to be non-relevant anymore and CLI works perfectly in this case. I tested it with:
`gh pr list -a vilmibm -l blocked,enhancement` and it gave me this PR https://github.com/cli/cli/pull/3557 as a result.
It was green just because logic error(it should have been `if err == nil || err.Error() != "..."` instead of `if err == nil && err.Error() != "..."`)